docs/jsdoc-style-guide.md

```diff
@@ -0,0 +1,38 @@
+---
+id: jsdoc-style-guide
```

Comment:
Is this id, title table needed? I have removed the table in dev concepts docs and only kept the title. We can either have it in all docs or just have title, for consistancy.

Reply:
Yes, you're correct, that stuff should be removed